vnext/Microsoft.ReactNative/ReactHost/CrashManager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "CrashManager.h"
+#include "ReactInstanceWin.h"
+
+#include <winrt/Windows.Storage.h>
+
+#include <WerApi.h>
+
+namespace Mso::React {
+
+// when calling SetUnhandledExceptionFilter the previous filter is returned
+// this should be saved so that if needed it can be called after the new unhandled exception has finished executing
+// this allows the Windows Error Reporting system to process the error and upload the data for processing
+LPTOP_LEVEL_EXCEPTION_FILTER g_previousExceptionFilter = nullptr;
```

Comment:
>g_previousExceptionFilter [](http://example.com/codeflow?start=29&length=25)

It is better to use the `static` keyword for the variables specific only to this cpp file. It helps to avoid any unexpected linker conflicts.